### ISSUE TITLE
Add indication for blocked elements on a page

### ISSUE THREAD

**Issue by SwartzCr** [EFForg/privacybadger#688]
Sometimes Privacy Badger will block third party content and the user will never know that it was supposed to be there leading to sites being confusingly broken
It could be possible to represent blocked divs and list what the source for that div is so that users know what third parties to unblock to make the site work
Imagine a toggleable option where for every div that PB blocks the resource of it adds a red dot or other indication, where in the mouseover text for that option it lists the third-party URL
This might be something we want to add for 2.0/3.0

**Comment by cooperq** [comment]
:+1: I think this is a good idea, but I do think we should think carefully about how such a UI is implemented. It has the danger of being extremely ugly if done wrong. We should also see if any of the other ad/tracker blockers do something similar for ideas about how to implement it.

**Comment by ghostwords** [comment]
We would like Privacy Badger to become an install-and-forget kind of extension. Anything that gets into customizing sliders needs to be clearly worth it benefit/complexity-wise. This proposal seems complex, while we already have a recommended alternative for when sites break, which is to use the "Disable Privacy Badger for This Site" button. Closing as "wontfix".
